src/resources/bank_account.rs

```diff
@@ -42,7 +42,7 @@ pub struct BankAccount {
 
     /// Whether this bank account is the default external account for its currency.
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub default_for_currency: Option<Currency>,
+    pub default_for_currency: Option<bool>,
```

Comment:
Because this file is auto-generated, instead of making this change we should rather fix the code-generator to handle this case correctly; otherwise this change would get smashed next time codegen gets run.